packages/easy/src/types/Uri.ts

```diff
@@ -61,6 +62,10 @@ export class EasyUri implements Uri {
     return toRoute(this.host, this.resource, ...this.segments);
   }
 
+  get isInternal (): boolean  {
+    return toRoute(this.host) === ctx.env.host;
```

Comment:
The host part of the uri has a fallback to `$host`, do we need that here as well?
```suggestion
    return toRoute(this.host) === ctx.env.host ?? '$host';
```